### DISCUSSION TITLE
Duda configuracion .env y .env-example

### DISCUSSION THREAD

**Discussion by germansalinas1994** [frlp-utn-ingsoft/catedra#101] [Q&A] [answered]
Buenas tardes, quería consultar sobre los archivos .env y .env-example 
Entendemos que para utilizarlo tenemos que instalar la libreria "python-dotenv", por lo que también debería incluirse en el archivo requirements.txt que está incluido y lo utilizaremos en el dockerfile.
Se supone que en el .env tenemos que  incluir las credenciales reales de la base de datos y datos sensibles, por lo tanto deberiamos agregarlo en el gitignore y dockerignore? 
El archivo .env-example deberia mostrar ejemplo de lo que son las variables de entorno que deberían configurarse y no debería ser agregado en el gitignore y dockerignore

**Comment by rodriguezemautn** [comment, marked as answer]
Si vas a incluir variables de entorno que luego vas a leer en el código de la aplicación si es necesario hacer lo que vos indicas. 
Si en el .env solo lo vas a consumir en el Dockerfile e imagen no seria necesario.
El archivo env-example no lleva . al comienzo; es solo un template que construye el desarrollador ya que como vos decís  el .env no debe ser subido al repo de git ni a las registry de contenedores.
